README.md

```diff
@@ -173,6 +173,42 @@ Here is an example config file from the gnatcov project:
 }
 ```
 
+# Integration with LanguageClient-Neovim
+
+If you want to integrate the ada language server into Neovim, you can use the
+[LanguageClient-neovim](https://github.com/autozimu/LanguageClient-neovim).
+
+You'll have to [install](#install) the ada language server manually somewhere on your
+computer, and then you can add the following line to your `init.vim` file:
+
+```viml
+" replace the path below with the proper path to the ada_language_server executable
+let g:LanguageClient_serverCommands = {
+    \ 'ada': ['path/to/ada_language_server'],
+    \ }
+" if you already have LanguageClient_serverCommands, just add a line for ada.
+```
+
+To configure the ALS for a specific workspace/project, you can use the
+`.vim/settings.json` file. It is mandatory as soon as you want to use a
+specific `.gpr` project file.
+
+This is the way to specify a project file, eg. you cannot open a project file
```

Comment:
will ada_language_server be launched in the same directory as vim? If so, I guess we could say that, if there's a single .gpr present, then it will be used, otherwise a project should be explicitly given in .vim/settings.json with (...) ?

Reply:
I don't know (didn't check), but it isn't documented as such (ie. with what the server is spawned as cwd isn't documented), so I would be wary of documenting that.

Reply:
Looking at the Rust sources for the vim plugin, the language server is spawned in `src/language_server_protocol.rs:2956`. It seems to use a variable `rootPath` as the current directory. This variable is extracted from the parameters (which I assume can be set in the vim configuration, but I haven't found that yet). If none is provided, it is computed via the function `getRootPath` in `src/utils.rs`. The latter has a number of hard-coded algorithm to compute this path depending on the language.  For Ada, it would default to the directory containing `.git`, `.hg` or `.svn`

Reply:
See also https://github.com/neoclide/coc.nvim/issues/107

One interesting comment there is `It is configurable, use g:rooter_patterns, check out :h g:rooter_patterns, same as vim-rooter.` which maybe is also related to how the `.vim` directory is found.